### PR TITLE
I added an r script @johanneskopton

### PR DESCRIPTION
**Pull request by johanneskopton**
Solve #1.

### CODE REVIEW

File: R_script.R [hunk 1]
```diff
@@ -0,0 +1 @@
+1+1
```
Comment (CWWhitney) [inline R_script.R:1]: This is great :)

Comment (CWWhitney) [reply R_script.R:1]: @johanneskopton what do you think?

Comment (johanneskopton) [reply R_script.R:1]: it surely is great

**Review by CWWhitney — APPROVED** [review]
Looks good to me - if you like it please approve pull request @johanneskopton

**Pull request merged**